app/assets/stylesheets/overrides/sufia.css.scss

```diff
@@ -0,0 +1,22 @@
+#home_header {
+  background: image_url('home_bg.jpg') center 100% no-repeat;
```

Comment:
Name of function `image_url` should be written in all lowercase letters with hyphens instead of underscores<br>Prefer double-quoted strings